### PR TITLE
added Flask-MonitoringDashboard

### PR DESCRIPTION
**Pull request by FlyingBird95**

### CODE REVIEW

File: flask_website/listings/extensions.py [hunk 1]
```diff
@@ -683,7 +683,22 @@ def docserver(self):
...
         github='rbw0/flask-snow'
-    ),    
+    ),
+    Extension('Flask-MonitoringDashboard', 'Patrick Vogel',
+        description='''
+            <p>
+              Flask Monitoring Dashboard is an extension that offers 3 main functionalities with little effort from the 
+              Flask developer:
+              <ol>
+                <li>Monitor the Flask application</li>
```
Comment (mircealungu) [inline flask_website/listings/extensions.py]: Monitor the **performance of a** Flask application?

File: flask_website/listings/extensions.py [hunk 2]
```diff
@@ -683,7 +683,22 @@ def docserver(self):
...
+                <li>Monitor the test coverage</li>
+                <li>Collect extra information about slow requests (so called outliers)</li>
```
Comment (mircealungu) [inline flask_website/listings/extensions.py]: Collect extra information about request outliers (requests which are slower than normal)

**Comment by miguelgrinberg** [comment]
This issue will be automatically closed due to being inactive for more than six months. Please reopen if you need more assistance.

**Comment by miguelgrinberg** [comment]
Sorry for the issue noise... was doing some batch clean up on my own issues and a few Flask ones slipped.

**Comment by FlyingBird95** [comment]
Hi @miguelgrinberg ,

thanks for keeping this PR open. I would like to ask you what should be changed before this PR can be accepted?

Thanks in advance,

Patrick

**Comment by miguelgrinberg** [comment]
@FlyingBird95 I'm not involved with the Flask website, so I do not know what the criteria is for accepting PRs.

**Comment by FlyingBird95** [comment]
@miguelgrinberg then who is?

**Comment by davidism** [comment]
This website is no longer maintained. The docs are moving to Read the Docs, and the home page is moving to our Lektor static site. I may in the future move the extension list to Lektor as well, which is why I'm leaving these open for reference, but I'm not going to merge them or make a new deploy of this site.

**Pull request merged**